CMP1903_A1_2324/Program.cs

```diff
@@ -14,6 +14,12 @@ static void Main(string[] args)
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
              */
```

Comment:
successfully creates the game and plays it. Test classed used instead of object however the results will be the same